lib/codegen/model.ejs

```diff
@@ -24,13 +24,18 @@ function printMethod(modelName) {
     return method + params.join(', ') + ') {';
 }
 
+function joinParams(params) {
+  const newParams = params.map(param => `${param}.toJSON()`);
```

Comment:
Not all parameters need to be converted to json. `toJSON` is not always available.